### PR TITLE
Translation for Duo 64/256m, Pioneer and CH5*

### PR DESCRIPTION
**Pull request by wychlw**

### CODE REVIEW

File: 100ASK/README.md
Comment (Sebastianhayashi) [inline 100ASK/README.md:1]: All about "hardware development board information" or any other describe about board information should use "hardware information" instead
the correct one should goes like:
Hardware Development Board Information -> Hardware Information

File: BeagleV-Ahead/README.md
Comment (Sebastianhayashi) [inline BeagleV-Ahead/README.md:1]: same here

File: BeagleV-Fire/README.md
Comment (Sebastianhayashi) [inline BeagleV-Fire/README.md:1]: same here

File: CH32V103/README.md
Comment (Sebastianhayashi) [inline CH32V103/README.md:1]: same here

File: CH32V203/README.md
Comment (Sebastianhayashi) [inline CH32V203/README.md:1]: same here

File: CH32V208/README.md
Comment (Sebastianhayashi) [inline CH32V208/README.md:1]: same here

File: CH32V303/README.md
Comment (Sebastianhayashi) [inline CH32V303/README.md:1]: same here

File: CH32V305/README.md
Comment (Sebastianhayashi) [inline CH32V305/README.md:1]: same here

File: CH32V307/README.md
Comment (Sebastianhayashi) [inline CH32V307/README.md:1]: same here

File: 100ASK/Melis/README.md [hunk 1]
```diff
@@ -46,36 +46,36 @@ KERNEL=="aw_efex[0-9]*", MODE="0666"
...
+System should boot up normally, with output visible through the serial port.
 
-## 实际结果
+## Actual Results
 
-系统正常启动，成功通过串口查看输出。
+System booted up successfully, and output was visible through the serial port.
 
-### 启动信息
+### Boot Information
```
Comment (Sebastianhayashi) [inline 100ASK/Melis/README.md]: Boot Information -> Boot Log

File: 100ASK/Melis/README_zh.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## 实际结果
+
+系统正常启动，成功通过串口查看输出。
+
+### 启动信息
+
+屏幕录像：
+
+```log
```
Comment (Sebastianhayashi) [inline 100ASK/Melis/README_zh.md]: log missing

Comment (KevinMX) [reply 100ASK/Melis/README_zh.md]: This is expected: we DO NOT have this board right now

**Pull request merged**